lib/_stream_writable.js

```diff
@@ -198,7 +199,7 @@ ObjectDefineProperty(WritableState.prototype, 'buffer', {
   get: internalUtil.deprecate(function writableStateBufferGetter() {
     return this.getBuffer();
   }, '_writableState.buffer is deprecated. Use _writableState.getBuffer ' +
-     'instead.', 'DEP0003')
+    'instead.', 'DEP0003')
```

Comment:
Could you also reverse this, it looks irrelevant to the PR?

Reply:
@lundibundi : Taken care.